SensorController/Core/Inc/System/OS_Ctrl.h

```diff
@@ -18,6 +18,7 @@
 #include "semphr.h"
 /* Macros */
 // Task Config
+#define WATCHDOG_PRI	tskIDLE_PRIORITY + 3
```

Comment:
Lets make this ``` configMAX_PRIORITIES - 1 ```

So the watch always as the highest priority

Reply:
sounds good